numpy/f2py/crackfortran.py

```diff
@@ -2985,7 +2983,7 @@ def expr2name(a, block, args=[]):
 
 def analyzeargs(block):
     setmesstext(block)
-    implicitrules, attrrules = buildimplicitrules(block)
+    implicitrules = buildimplicitrules(block)[0]
```

Comment:
```suggestion
    implicitrules, _ = buildimplicitrules(block)
```
Is probably slightly clearer